test/plugin/test_in_tail.rb

```diff
@@ -2110,8 +2111,10 @@ def test_EACCES_error_after_setup_watcher
       assert($log.out.logs.any?{|log| log.include?("stat() for #{path} failed with Errno::EACCES. Drop tail watcher for now.\n") })
     end
   ensure
-    FileUtils.chmod(0755, "#{TMP_DIR}/noaccess")
-    FileUtils.rm_rf("#{TMP_DIR}/noaccess")
+    if File.exist?("#{TMP_DIR}/noaccess")
+      FileUtils.chmod(0755, "#{TMP_DIR}/noaccess")
+      FileUtils.rm_rf("#{TMP_DIR}/noaccess")
+    end
   end unless Fluent.windows?
```

Comment:
I'm not sure, but is this `if ... end` needed?
It seems that `omit` is enough.

Reply:
If it is needed, could you explain in the commit message, too?

Reply:
It's needed because It's in ensure block. It's always called.
I think writing it in commit message is too verbose.

Reply:
Sorry, I've overlooked `ensure` block behavior, so no need to too verbose explanation.